lib/gpu/lib/src/render_pass.dart

```diff
@@ -10,14 +10,15 @@ base class ColorAttachment {
   ColorAttachment({
     this.loadAction = LoadAction.clear,
     this.storeAction = StoreAction.store,
-    this.clearValue = const ui.Color(0x00000000),
+    // TODO(bdero): Why can't vm.Vector4 constructors be const?
+    vm.Vector4? clearValue = null,
```

Comment:
Vector4 (and all vector math classes) are backed by a fixed size TypedData array that aren't const constructable.

Reply:
Meh, oh well. :/ Thanks for taking a look.